packages/cms-base-layer/app/components/public/cms/element/CmsElementImage.vue

```diff
@@ -27,7 +26,7 @@ const {
 } = useCmsElementImage(props.content);
 
 const DEFAULT_THUMBNAIL_SIZE = 10;
-const imageElement = useTemplateRef("imageElement");
+const imageElement = ref<HTMLImageElement>();
```

Comment:
Why did you change from templateRef to ref?

Reply:
typecheck was failing due to incompatibility between useTemplateRef and the old @nuxt/image module. Now it's like expected -  once I've upgraded to v 2.0 of the module 💪🏼